src/CMake/FindVisItPython.cmake

```diff
@@ -112,165 +116,164 @@ endif()
 
 find_package(PythonInterp REQUIRED)
 if(PYTHONINTERP_FOUND)
+    message(STATUS "PYTHON_EXECUTABLE ${PYTHON_EXECUTABLE}")
```

Comment:
Comment above says this is from Conduit. Seems like its a better fit in BLT. Doesn't BLT need to basically do this too? Not suggesting you should add to BLT. Its more of a question for @cyrush and whether it should be added to BLT.